beef/beef_tx_test.go

```diff
@@ -51,51 +52,49 @@ func TestDecodeBEEF_DecodeBEEF_HappyPaths(t *testing.T) {
 				},
 				Transactions: []*TxData{
 					{
-						Transaction: &bt.Tx{
+						Transaction: &trx.Transaction{
 							Version:  1,
 							LockTime: 0,
-							Inputs: []*bt.Input{
+							Inputs: []*trx.TransactionInput{
```

Comment:
this `trx` do not suggest that we are working on Transaction ? as to me `trx.Input` if not used alias u would have 
`transation.Input` with ur naming u would have `transaction.TransactionInput`

Reply:
Okay, I think that I understand - then I replaced trx with sdk (those are models from sdk) please tell me if it's ok.